content/before-you-fly/code-of-conduct.md

```diff
@@ -6,46 +6,45 @@ weight: 2
 featured_image: "/img/become-a-member.jpg"
 ---
 
-The Blue Mountains Soaring Club thanks you for your membership contribution. We also welcome your ideas and the giving of your time to club meetings and projects. Your club supports your access to sites but needs your support in return.
+The Blue Mountains Soaring Club (BMSC) thanks you for your membership contribution. We also welcome your ideas and the giving of your time to club meetings and projects. Your club supports your access to sites but **needs your support in return**.
 <!--more-->
 
-The Blue Mountains Soaring Club is an affiliate member of the [Sports Aviation Federation of Australia (SAFA)](https://www.safa.asn.au/) and as such the BMSC adopts the SAFA Code of Conduct. SAFA Code of Conduct can be found at; https://members.safa.asn.au/isonic-downloaddoc.php?docid=0X0X1X1X47Wnkvb2IwaDFScW9iOU9nVi9Ody9sdz09
+The Blue Mountains Soaring Club is an affiliate member of the [Sports Aviation Federation of Australia (SAFA)](https://www.safa.asn.au/) and as such the **BMSC adopts the SAFA Code of Conduct**. You can [read the SAFA Code of Conduct on their website](https://members.safa.asn.au/isonic-downloaddoc.php?docid=0X0X1X1X47Wnkvb2IwaDFScW9iOU9nVi9Ody9sdz09).
 
-As all NSWPHA pilots are also members of the New South Wales Hang gliding and Paragliding Association, we also remind you to acknowledge its Member code of Behaviour; http://www.nswpha.org/policies
+As all NSW pilots are also members of the [New South Wales Hang gliding and Paragliding Association (NSWHPA)](https://www.nswhpa.org/), we also remind you to acknowledge its [Member code of Behaviour](https://www.nswhpa.org/s/NSWHPA-Member-Code-of-Behaviour-2019.pdf).
```

Comment:
I think you meant "NSW pilots" rather than NSWHPA pilots (and note the typo - you had NSWPHA here both in the third word as well as the link (which broke the link). When fixing the link I also updated it to link directly to their code of behaviour? Let m know if you prefer to link just to the policies.